Sources/LiUIComponents/NewMessage/LiReplyViewController.swift

```diff
@@ -35,15 +35,11 @@ class LiReplyViewController: LiBaseNewMessageViewController {
         setupToolbar()
     }
     func setupNavigationController() {
-        let navBar: UINavigationBar = UINavigationBar(frame: CGRect(x: 0, y: 0, width: view.frame.size.width, height: 64))
-        self.view.addSubview(navBar)
-        let navItem = UINavigationItem(title: LiHelperFunctions.localizedString(for: "Reply"))
+        title = LiHelperFunctions.localizedString(for: "Reply")
```

Comment:
it's fine for now, but as a future practise I think it makes sense to reference instance variables with self.
It serves two purposes:
1: It's easier to identify the instance vars from the local vars
2: It's more consistent especially when you're using closures or accidentally declair a var with the same name

Reply:
Let's discuss this offline.